### PR TITLE
126 dataset paired loader

### PR DESCRIPTION
**Pull request by agrimwood**
# Description

unit test file for paired loader added: /test/unit/test_paired_loader.py
comments amended for clarity in: /deepreg/dataset/loader/paired_loader.py

Fixes # 126

## Type of change
unit test

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- unit test ran successfully on local machine.

**Test Configuration**:

- DeepReg CPU conda env
- Windows 10
- main branch from 14 / 07 / 2020

# Checklist:

- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings
- [ x] I have added tests that prove my fix is effective or that my feature works
- [ x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by NMontanaBrown** [comment]
Hi @agrimwood, there have been a few merges to master - can you update your local branch to reflect these new changes prior to review?

File: test/unit/test_paired_loader.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+Tests functionality of the PairedDataLoader
+"""
+import pytest
+
+from deepreg.dataset.loader.h5_loader import H5FileLoader
+from deepreg.dataset.loader.paired_loader import PairedDataLoader
+
+# assign values to input vars
+data_dir_path = "./data/test/h5/paired/test"
+sample_label = "sample label"
```
Comment (mathpluscode) [inline test/unit/test_paired_loader.py:11]: the value should be "sample" or "all", check `get_label_indices` in `deepreg/dataset/util.py`

File: test/unit/test_paired_loader.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+seed = 42
+moving_image_shape_arr = (999, 999, 999)
```
Comment (mathpluscode) [inline test/unit/test_paired_loader.py:13]: I'm wondering maybe smaller values will accelerate the tests?
I suggest (16, 16, 16) or (8, 8, 8)

File: test/unit/test_paired_loader.py [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+"""
+import pytest
+
+from deepreg.dataset.loader.h5_loader import H5FileLoader
+from deepreg.dataset.loader.paired_loader import PairedDataLoader
+
+# assign values to input vars
+data_dir_path = "./data/test/h5/paired/test"
+sample_label = "sample"
+seed = (8, 8, 8)
```
Comment (mathpluscode) [inline test/unit/test_paired_loader.py:12]: the seed should be int lol ? like 0 ?
and the shapes in the following should be (8, 8, 8)

**Comment by NMontanaBrown** [comment]
> just change the seed and shapes and we are good!

And pull from master again! 😅

**Comment by NMontanaBrown** [comment]
> Sorry for the delay!
> 
> most recent commit on 126:
> `seed` value `int 1` and inputs are now both tuples `(8,8,8)`

Hi @agrimwood thanks for addressing! Still need to update with master branch before merging. Can you pull from master and then @mathpluscode can approve for merge?

**Pull request merged**